spec/push/message_apns_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'spec_helper'
+
+describe Push::MessageApns do
+  let(:notification_options) {
+    {
+      app: 'com.test.app',
```

Comment:
I think it would be worth generating some random values for these and referring to them separately. That could either be in separate `let` statements for each element, or just within this hash and refer to them below as `notification_options[:app]` etc.